api/helpers.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	"fmt"
+	"github.com/ledgerwatch/diagnostics"
+	"github.com/pkg/errors"
```

Comment:
do you see any advantage of using `pkg/errors` over native errors.Is/errors.As/fmt.Printf("%w") introduced in go 1.13? https://go.dev/blog/go1.13-errors